### ISSUE TITLE
[BUG] IBM equal accessibility plugin hangs indefinitely for IBM Cloud ApiDocs application

### ISSUE THREAD

**Issue by npovatho** [IBMa/equal-access#2417]
**Describe the bug**

The Accessibility checker tools hangs on our applications

**To Reproduce**
1. Go to 'https://cloud.ibm.com/apidocs/secrets-manager/secrets-manager-v2'
2. Go to F12 to the IBM Equal Accessibility Checker Tool
3. Click on Scan
4. See that it hangs & never gets us the scan result

**Expected behavior**
We should be seeing the accessibility test results.

**Screenshots**
NA

**Desktop (please complete the following information):**
 - OS:Mac Sonoma
 - Browser Chrome
 - Version 42.0.7444.60

**Additional context**
Additional Urls : 

In production https://cloud.ibm.com/apidocs/vpc/latest#update-subnet-reserved-ip
In development https://dev.console.test.cloud.ibm.com/docs/apis/secrets-manager/secrets-manager-v2?code=curl

## Definition of Done

- [ ] Design artifacts reviewed
- [ ] Work item implemented
- [ ] Automated test case created (when applicable)
- [ ] Unit test completed
- [ ] Run Checker (when applicable)
- [ ] Manual and Screen Reader Testing (when applicable)
- [ ] Video recording of user experience walkthrough (if impact to UX)
- [ ] PR created

**Comment by tombrunet** [comment]
It's generally a large page. If I disable `element_tabbable_visible` and `style_focus_visible`, it completes in about three minutes. Still slow, but does complete. Will need to take a deep dive into these two rules in particular, but need to spend some time looking at overall rule performance to see where else we can improve.
